templates/Toolkit template/SoftwareName_Toolkit/SoftwareName_Adapter/SoftwareNameAdapter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BH.Adapter;
+using BH.Engine.$ext_safeprojectname$;
+
+namespace BH.Adapter.$ext_safeprojectname$
+{
+    public partial class $ext_safeprojectname$ : BHoMAdapter
```

Comment:
Think this need to be `$ext_safeprojectname$Adapter` rather than just `$ext_safeprojectname$`